examples/digital-asset-manager/src/components/Assets.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import Loader from "./Loader";
+import CatalogItem from "./CatalogItem";
+import AssetCount from "./AssetCount";
+
+interface ImageData {
+  name: string;
+  thumbnails: {
+    mini: string;
+    thul: string;
+    webimage: string;
+  };
+  property_extension: string;
+}
+
+const Assets = ({
+  searchTerm,
+  collectionId,
+}: {
+  searchTerm: string;
+  collectionId: string;
+}) => {
+  const [loading, setLoading] = useState(true);
+  const [assetUrls, setAssetUrls] = useState<ImageData[]>([]);
+  const [assetCount, setAssetCount] = useState(0);
+
+  const listFiles = () => {
```

Comment:
This function also fetches the files from a netlify function so i don't feel like it's name captures all the responsibilities